server/core/src/main/java/org/infinispan/server/core/ServerManagement.java

```diff
@@ -14,4 +15,13 @@
    void stop();
 
    Set<String> cacheManagerNames();
+
+   DefaultCacheManager getCacheManager(String name);
+
+   void ignoreCache(String cacheManager, String cache);
```

Comment:
suggestion: It could return a `CompletionStage<Boolean>` to signal if the operation was successful or not and, internally, use the `cache.putAsync()` method. 
Also, the `getCacheManager()` may not be needed if the method checks if the cache-manager and cache exist.